src/v2/Apps/Order/Components/__tests__/SavedAddresses.jest.tsx

```diff
@@ -21,209 +23,219 @@ class SavedAddressesTestPage extends RootTestPage {
   }
 }
 
-describe("Saved Addresses mutations", () => {
-  const { mutations, buildPage } = createTestEnv({
-    Component: (props: any) => (
-      <SavedAddressesFragmentContainer inCollectorProfile {...props} />
-    ),
-    defaultData: userAddressMutation,
-    TestPage: SavedAddressesTestPage,
-    query: graphql`
-      query SavedAddressesMutation_Test_Query {
-        me {
-          ...SavedAddresses_me
-        }
-      }
-    `,
+describe("Saved Addresses", () => {
+  beforeAll(() => {
+    ;(useTracking as jest.Mock).mockImplementation(() => ({
+      trackEvent: jest.fn(),
+    }))
```

Comment:
most of this file diff is noise from adding a top-level describe, so that all tests can take advantage of this `beforeEach`.